lib/src/settings_tile.dart

```diff
@@ -57,11 +55,10 @@ class SettingsTile extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final platform = Theme.of(context).platform;
-    if (platform.isIOS) {
+    if (platform.isIOS(context)) {
       return iosTile();
-    } else {
-      return androidTile();
     }
+    return androidTile();
```

Comment:
I think the same idea in [this comment](https://github.com/yako-dev/flutter-settings-ui/pull/41/files#r528966938) can be applied here as well.